server/controllers/app.controller.spec.js

```diff
@@ -224,6 +230,8 @@ describe('The app controller', function() {
     this.sandbox.stub(modulesMocks.redisClient, 'storeShopAsync');
     this.sandbox.stub(modulesMocks.redisClient, 'removeShopAsync');
     this.sandbox.stub(modulesMocks.redisClient, 'quitAsync');
+    this.sandbox.stub(modulesMocks.redisClient, 'getShopAsync')
+    .returns(Promise.resolve({ accessToken: '127424ab9aa0ebce26dfdc786bc7fba4'}));
```

Comment:
[The same](https://github.com/MakingSense/doppler-for-shopify/pull/196/files#r388378122) her.